core/src/main/java/io/questdb/cairo/GenericRecordMetadata.java

```diff
@@ -29,7 +29,11 @@
 
 public class GenericRecordMetadata extends AbstractRecordMetadata {
 
-    public static void copyColumns(RecordMetadata from, GenericRecordMetadata to) {
+    /**
+     * This method will throw when called on table writer metadata in case of deleted and re-created columns.
```

Comment:
Outdated comment? I don't see any `throw` in here.

Reply:
This place uses this method to copy metadata with deleted columns. And it doesn't throw because `add` is overridden.

https://github.com/questdb/questdb/blob/62bb9a05340656f9c3e93847907c45b0bd2bed5c/core/src/test/java/io/questdb/test/griffin/wal/fuzz/FuzzTransactionGenerator.java#L211

Reply:
The fact that `add` method is overridden in tests doesn't make `copyColumns` less error-prone. We have a bug around this method to be fixed by this PR and there is no guarantee that future code won't introduce something similar. I decided to leave a comment as a caution.

Improved the comment and the method name in e69608491f03597eeadca6df220be2f5a2ffd207